### PR TITLE
samples: sensor: lps22hh: add standalone sample

### PR DESCRIPTION
**Pull request by pabigot**
Although this sensor is demonstrated by the X-NUCLEO-IKS01A3 sample,
maintenance of the driver is simplified if it can be tested in
isolation.  Provide a sample modeled on hts221.

Signed-off-by: Peter Bigot <peter.bigot@nordicsemi.no>

### CODE REVIEW

File: samples/sensor/lps22hh/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Requirements
+************
+
+This sample uses the LPS22HH sensor controlled using the I2C interface.
+
+References
+**********
+
+ - LPS22HH: http://www.st.com/en/mems-and-sensors/lps22hh.html
```
Comment (dbkinder) [inline samples/sensor/lps22hh/README.rst]: Delete leading space (affects the rendered HTML appearance)

File: samples/sensor/lps22hh/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Building and Running
+********************
+
+ This project outputs sensor data to the console. It requires an LPS22HH
```
Comment (dbkinder) [inline samples/sensor/lps22hh/README.rst]: Delete leading space on these two lines (affects the rendered HTML appearance)

File: samples/sensor/lps22hh/README.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   :zephyr-app: samples/sensor/lps22hh
+   :board: nrf52_pca10040
+   :shield: x_nucleo_iks01a3
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+ .. code-block:: console
```
Comment (dbkinder) [inline samples/sensor/lps22hh/README.rst]: Delete leading space (affects the rendered HTML appearance)

**Review by dbkinder — DISMISSED** [review]
LGTM, after fixing a few lines with leading spaces

**Comment by dbkinder** [comment]
It's subtle, but if you look at https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/20803/docs/samples/sensor/lps22hh/README.html you'll see some of the text and code-blocks aren't left-aligned the same (because of the leading spaces)

**Comment by pabigot** [comment]
@dbkinder Please revisit this one too; your change requests have been completed.  Thanks.

**Comment by pabigot** [comment]
@dbkinder Please confirm the changes meet your expectations.  Thanks.

**Review by nashif — APPROVED** [review]

**Pull request merged**